### ISSUE TITLE
Feature Request: Minimize/Close to tray

### ISSUE THREAD

**Issue by cwoods71** [chrisknepper/android-messages-desktop#21]
Minimize and close to tray would be nice features.

**Comment by spncrhrstn** [comment]
Hey @chrisknepper I appreciate having the minimizing to tray functionality, but I feel it could be improved a little. In my experience with other apps, typically the app stays in the taskbar when the minimize window button is clicked, but if you close the window, it's removed from the taskbar but minimizes to the system tray (still running in the background). GPDMP is another good example of what I'm describing. The main reason I'm asking for this is because when I minimize Messages, I can't alt-tab back to the window because it's now in the system tray. 

Again this is my own opinion, I do appreciate the work you've put into the app. This is probably a Windows only thing, BTW.

**Comment by hexeth** [comment]
I agree. The best practice in my opinion would be for the following functionality:

Main Window Minimize = minimize
Main Window Close = close to tray, trigger one-time "application still running" notification 
Tray Icon Quit = close application

An option could always be added to toggle the tray behavior at a later date, but the above would be the best until that time.

**Comment by chrisknepper** [comment]
@hexeth @spncrhrstn Agree 100%. I'll need to read the Electron docs a bit more to make sure we only go to the tray specifically when the Close button is pressed, but I think this would make a better experience for sure.

**Comment by chrisknepper** [comment]
Check out v0.3.0 and let me know what you all think about the changes. If there's still something you'd like to see and it wasn't covered, please open a new issue.

**Comment by spncrhrstn** [comment]
Just installed the update, so far so good! Thanks for your work on this!
